src/graphql/root/mutation/twofa-generate.ts

```diff
@@ -0,0 +1,16 @@
+import { GT } from "@graphql/index"
+import TwoFAGeneratePayload from "@graphql/types/payload/twofa-generate"
+
+const TwoFAGenerateMutation = GT.Field({
+  type: GT.NonNull(TwoFAGeneratePayload),
+  resolve: async (_, __, { wallet }) => {
+    try {
+      const towFA = await wallet.generate2fa()
```

Comment:
Typo, should be `twoFA` instead of `towFA`

Reply:
Fixed.